src/app/signup/signup.component.ts

```diff
@@ -114,15 +117,16 @@ export class SignupComponent implements OnInit {
                           horizontalPosition:'right'
                         });
                       }
-                       else{                      
-                       confirm('Sorry, an error occurred. Please email support@digitaltaxusa.com');
+                       else{   
+
+                       confirm(environment.error_message);
                        
                     } 
                  });
                       }
                        else{
                           /*show error message as recaptcha thresold value not ok*/
-                          confirm('Sorry, an error occurred. Please email support@digitaltaxusa.com');
+                          confirm(environment.error_message);
```

Comment:
@Bithika-Redapple Instead off error_message as I am sure there will be more error messages. You could have labeled this as `default_error_message`. Something like that.